core/Cargo.toml

```diff
@@ -28,12 +28,11 @@ tokio = { version = "1.16", optional = true }
 wasm-bindgen-futures = { version = "0.4.19", optional = true }
```

Comment:
```suggestion
```

Reply:
remove in below in async-client to make the CI happy

Reply:
I noticed the `wasm-bindgen-futures` is used under `#[cfg(all(feature = "async-wasm-client", target_arch = "wasm32"))]` this feature.

If I remove it from `async-client` as you suggest, will this break any functionality? 

```sh
$ cargo check --release --features async-wasm-client --target wasm32-unknown-unknown
    Checking jsonrpsee-core v0.16.2 (/home/bear/coding/jsonrpsee/core)
error[E0433]: failed to resolve: use of undeclared crate or module `wasm_bindgen_futures`
   --> core/src/client/async_client/mod.rs:210:3
    |
210 |         wasm_bindgen_futures::spawn_local(async move {
    |         ^^^^^^^^^^^^^^^^^^^^ use of undeclared crate or module `wasm_bindgen_futures`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `jsonrpsee-core` due to previous error

```

Reply:
Okay. Let me have a try.

Reply:
don't do it, it's a mess

lemme find a branch for it instead.

Reply:
Nice, I'll revert these changes after your branch merged.